src/yunohost/backup.py

```diff
@@ -152,6 +152,21 @@ def _clean_tmp_dir(retcode=0):
         'hooks': {},
     }
 
+    # Initialize backup list
+    backup_csv_path = os.path.join(tmp_dir, 'backup.csv')
+    try:
+        backup_csv = open(backup_csv_path, 'a')
+    except:
+        logger.error(m18n.n('backup_csv_creation_failed'))
+
+    def add_into_list(source,dest):
+        try:
+            if dest.endswith("/"):
+                dest=dest+os.path.basename(source)
+            backup_csv.write("{0}\t{1}\n".format(source,dest))
```

Comment:
I use the TAB character as separator for the CSV. I am not usre it's a good idea (what if a path contain a tab ?)